tests/__snapshots__/HtmlReplacerFragmentTest__it_can_handle_text_with_an_outer_P_tag__1.txt

```diff
@@ -0,0 +1 @@
+<p>This is some fancy-<img src="https://twemoji.maxcdn.com/v/latest/72x72/1f483.png" alt="&#128131;" width="72" height="72" loading="lazy" class="twemoji"> Markdown/WYSIWYG text with surrounding <p> tags enabled. <img src="https://twemoji.maxcdn.com/v/latest/72x72/1f389.png" alt="&#127881;" width="72" height="72" loading="lazy" class="twemoji"></p></p>
```

Comment:
However with this fragment over here it's obvious `DOMDocument` executes the normal behavior. The `alt` attribute is incorrectly encoded to html entity.